.gitignore

```diff
@@ -4,3 +4,4 @@ diagnostic.test
 tests/*.diff
 spec/
 spec-runner/
+demo/sourcing.vars
```

Comment:
I would much rather just have this file committed instead of being generated by `demo/sourcing`.